### PR TITLE
Tasks 872-876: opus_xhosanavy, emotion

### PR DESCRIPTION
**Pull request by aklagoo**

### CODE REVIEW

File: tasks/task874_opus_xhosanavy_sr.json [hunk 1]
```diff
@@ -0,0 +1,704 @@
+{
+    "Contributors": [
+        "Archan Lagoo"
+    ],
+    "Source": [
+        "opus_xhosanavy (\"https://huggingface.co/datasets/opus_xhosanavy\")"
+    ],
+    "Definition": "In this task, you are given a sentence or sub-sentence. You must find the primary subject in the supplied sentence. The output must be a sub-sequence in the sentence.",
```
Comment (danyaljj) [inline tasks/task874_opus_xhosanavy_sr.json]: You can drop this: ""In this task, you are given a sentence or sub-sentence. You must " and simply say, "Find the ... ".

Comment (danyaljj) [reply tasks/task874_opus_xhosanavy_sr.json]: Also, can a given input have multiple primary subjects?

File: tasks/task875_emotion_classification.json [hunk 2]
```diff
@@ -0,0 +1,19417 @@
...
+            "input": "i am ever feeling nostalgic about the fireplace i will know that it is still on the property",
+            "output": "love",
+            "explanation": "The word \"nostalgic\" could either indicate \"love\" or \"sadness\", but the fact that it is still on the property eliminates \"love\"."
+        },
+        {
+            "input": "i am feeling grouchy",
+            "output": "anger",
+            "explanation": "The word \"grouchy\" clearly indicates the emotion \"anger\"."
+        }
+    ],
```
Comment (danyaljj) [inline tasks/task875_emotion_classification.json]: Please include more positive and negative examples (ideally, one from each emotion).

File: tasks/task876_emotion_esd.json [hunk 3]
```diff
@@ -0,0 +1,644 @@
+{
+    "Contributors": [
+        "Archan Lagoo"
+    ],
+    "Source": [
+        "emotion (\"https://huggingface.co/datasets/emotion\")"
+    ],
+    "Definition": "This task involves detection of phrases in sentences that evoke a specific emotion when together. This is meant to imitate human attention when reading sentences. The input consists of a sentence (string) and an emotion label (string) as \"SENTENCE\nLABEL\", while the ouptut must contain a series of phrases (string) that evoke the emotion formatted as \"phrase\nphrase\n...\".The list of emotions includes sadness, joy, love, anger, fear, and surprise.",
```
Comment (danyaljj) [inline tasks/task876_emotion_esd.json]: The definition is quite verbose. Wondering if you can shorten it without losing the key points.

File: tasks/task876_emotion_esd.json [hunk 4]
```diff
@@ -0,0 +1,644 @@
...
+        "emotion (\"https://huggingface.co/datasets/emotion\")"
+    ],
+    "Definition": "This task involves detection of phrases in sentences that evoke a specific emotion when together. This is meant to imitate human attention when reading sentences. The input consists of a sentence (string) and an emotion label (string) as \"SENTENCE\nLABEL\", while the ouptut must contain a series of phrases (string) that evoke the emotion formatted as \"phrase\nphrase\n...\".The list of emotions includes sadness, joy, love, anger, fear, and surprise.",
+    "Categories": [
+        "Emotive Sequence Detection"
+    ],
+    "Positive Examples": [
+        {
+            "input": "i feel frustrated when i have new music and new lyrics that clearly have nothing to do with each other\nanger",
+            "output": "frustrated\nclearly have nothing to do with each other",
```
Comment (danyaljj) [inline tasks/task876_emotion_esd.json]: I am not sure why we have "Clearly have nothing to do with each other" here. Shouldn't the output be one of the six labels?

File: tasks/task874_opus_xhosanavy_sr.json (hunk 1 above)
Comment (aklagoo) [reply tasks/task874_opus_xhosanavy_sr.json]: Not currently. The current task classifies one subject as the primary subject. In the cases where there are multiple subjects presented as a group ("P1, P2, P3, and P4"), I return the entire noun phrase.

File: tasks/task876_emotion_esd.json (hunk 3 above)
Comment (danyaljj) [inline tasks/task876_emotion_esd.json]: Does not add anything to the task definition: "This is meant to imitate human attention when reading sentences. "  drop it.

**Comment by danyaljj** [comment]
Please make sure to set the `"Categories": [""]` and `"Domains": [""]` fields according to [the taxonomy presented here](https://github.com/allenai/natural-instructions-expansion/blob/master/doc/task-hierarchy.md).

**Comment by aarunku5** [comment]
> Please make sure to set the `"Categories": [""]` and `"Domains": [""]` fields according to [the taxonomy presented here](https://github.com/allenai/natural-instructions-expansion/blob/master/doc/task-hierarchy.md).

@aklagoo can you please address this?

**Comment by aklagoo** [comment]
Really sorry about this. Will make all changes by the end of the day

**Comment by danyaljj** [comment]
- Dropped  `task876_emotion_esd`: if anyone is willing to improve it, we can consider it in another PR. 
 - Updated the language tags

**Pull request merged**